test/postControllerTest.js

```diff
@@ -0,0 +1,187 @@
+/* global it, describe */
+const express = require('express');
+const request = require('supertest');
+const postController = require('../controllers/postController');
+const sinon = require('sinon');
+const postService = require('../services/postService');
+const assert = require('assert');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(express.static('../public'));
+
+app.use('', postController);
+
+describe('postController', () => {
+  let stubGetAllPosts = sinon.stub();
+  let stubGetOnePost = sinon.stub();
+  let stubCreateNewPost = sinon.stub();
+
+  beforeEach(() => {
+    stubGetAllPosts = sinon.stub(postService, 'getAllPosts').resolves('expected');
+    stubGetOnePost = sinon.stub(postService, 'getOnePost').resolves('expected');
+    stubCreateNewPost = sinon.stub(postService, 'createNewPost').returns(Promise.resolve('expected'));
```

Comment:
Why `returns(Promise.resolve('expected'))` over `resolves('expected')`?

Reply:
Forgot to change back. Took awhile for me to get promises to work.